management/server/http/api/openapi.yml

```diff
@@ -1176,6 +1182,24 @@ components:
             - id
             - network_type
         - $ref: '#/components/schemas/RouteRequest'
+    Resource:
+      type: object
+      properties:
+        id:
+          description: Resource ID
```

Comment:
We should be more clear here what the ID is. In this case it is not the object ID but the ID of the actual resource we are targeting